tracer/test/Datadog.Trace.Tests/Telemetry/Collectors/ConfigurationTelemetryCollectorTests.cs

```diff
@@ -233,6 +233,26 @@ public void ConfigurationDataShouldReportDefaultValues()
         GetLatestValueFromConfig(data, "DD_TRACE_HEADER_TAGS", ConfigurationOrigins.Default).Should().Be(string.Empty);
         GetLatestValueFromConfig(data, "DD_LOGS_INJECTION", ConfigurationOrigins.Default).Should().Be(false);
         GetLatestValueFromConfig(data, "DD_TRACE_SAMPLE_RATE", ConfigurationOrigins.Default).Should().Be(1.0);
+        GetLatestValueFromConfig(data, "instrumentation_source", ConfigurationOrigins.Default).Should().Be("unknown");
+    }
+
+    [Fact]
+    public void ConfigurationDataShouldReportSSIValues()
+    {
+        var collector = new ConfigurationTelemetry();
+        var source = new NameValueConfigurationSource(new NameValueCollection
+        {
+            { "DD_INJECTION_ENABLED", "tracer" },
+            { "DD_INJECT_FORCE", "true" }
+        });
+
+        _ = new ImmutableTracerSettings(new TracerSettings(source, collector));
```

Comment:
It looks like your fork is pretty old, here's an updated initialization that matches the updated test code in `master`:

```suggestion
        _ = new TracerSettings(source, collector, new OverrideErrorLog());
```